services/workflows-service/src/customer/customer.repository.ts

```diff
@@ -75,30 +62,30 @@ export class CustomerRepository {
     });
   }
 
-  async findByApiKey<T extends Omit<Prisma.CustomerFindFirstArgs, 'where'>>(
-    apiKey: string,
-  ): Promise<CustomerWithProjects | null> {
-    return this.prisma.customer.findFirst({
-      where: {
-        authenticationConfiguration: {
-          path: ['authValue'],
-          equals: apiKey,
-        },
-      },
-      select: {
-        id: true,
-        name: true,
-        authenticationConfiguration: true,
-        displayName: true,
-        logoImageUri: true,
-        faviconImageUri: true,
-        customerStatus: true,
-        country: true,
-        language: true,
-        projects: true,
-      },
-    });
-  }
+  // async findByApiKey<T extends Omit<Prisma.CustomerFindFirstArgs, 'where'>>(
+  //   apiKey: string,
+  // ): Promise<CustomerWithProjects | null> {
+  //   return this.prisma.customer.findFirst({
+  //     where: {
+  //       authenticationConfiguration: {
+  //         path: ['authValue'],
+  //         equals: apiKey,
+  //       },
+  //     },
+  //     select: {
+  //       id: true,
+  //       name: true,
+  //       authenticationConfiguration: true,
+  //       displayName: true,
+  //       logoImageUri: true,
+  //       faviconImageUri: true,
+  //       customerStatus: true,
+  //       country: true,
+  //       language: true,
+  //       projects: true,
+  //     },
+  //   });
+  // }
```

Comment:
Dead code - make sure to delete it

Reply:
oh no i just set it here in order to test data migration so i need to seed the previous customer data